exonum/src/node/consensus.rs

```diff
@@ -924,7 +924,7 @@ impl NodeHandler {
             round,
             propose_hash,
             block_hash,
-            self.system_state.current_time(),
+            self.system_state.current_time().into(),
```

Comment:
Can we change the `SystemStateProvider` trait definition?

Reply:
Checked, no. Internals a tangled with `SystemTime` heavily, rewriting the whole thing will waste a lot of time without much profit.